File: Algo/getAllPrimeNumber.py
#Given a number N, N>=2. Print all prime numbers <= N.

def CheckPrime(num):
    IsItPrime = True
    for i in range(2,num):
        if num%i == 0:
            IsItPrime = False
            break
    
    return IsItPrime          
        
def GetPrimeNumbers(N):
    arrayOfPrime = []
    i = 2
    while i <= N:
        if CheckPrime(i) == True:
            arrayOfPrime.append(i)
        i +=1        
    return arrayOfPrime

if __name__ == "__main__":
    N = int(raw_input().strip())
    print GetPrimeNumbers(N)

File: Algo/hr_warmups/getAllPrimeNumber.js
//Given a number N, N>=2. Print all prime numbers <= N.

class Prime {
    getPrimeNumber(n) {
        var arrayOfPrime = [];
        var i = 2;
        while (n >= i) {
            if (this.checkPrime(i)) {
                arrayOfPrime.push(i);
            }
            i++;
        }
        return arrayOfPrime;
    }

    checkPrime(n) {
        var isItPrime = true;
        for (var i = 2; i < n; i++) {
            if (n % i == 0) {
                isItPrime = false;
                break;
            }
        }
        return isItPrime;
    }
}

var p = new Prime();
console.log(p.getPrimeNumber(10));